flink-runtime/src/main/java/org/apache/flink/runtime/metrics/util/MetricUtils.java

```diff
@@ -62,7 +64,7 @@
  */
 public class MetricUtils {
 	private static final Logger LOG = LoggerFactory.getLogger(MetricUtils.class);
-	private static final String METRIC_GROUP_STATUS_NAME = "Status";
+	public static final String METRIC_GROUP_STATUS_NAME = "Status";
```

Comment:
where is this used outside of this class?

Reply:
This is wrong. Will correct it.